knx/dpt/types.go

```diff
@@ -31,24 +32,106 @@ func (sw Switch) String() string {
 	if sw {
 		return "On"
 	}
-
 	return "Off"
 }
 
-// ValueTemp is DPT 9.001.
+// TrueFalse DPT 1.002
+type TrueFalse bool
+
+// Pack the datapoint value.
+func (b TrueFalse) Pack() []byte {
+	return packB1(bool(b))
+}
+
+// Unpack the datapoint value from the given data.
+func (b *TrueFalse) Unpack(data []byte) error {
+	return unpackB1(data, (*bool)(b))
+}
+
+// String generates a string representation.
+func (sw TrueFalse) String() string {
+	if sw {
+		return "True"
+	}
+	return "False"
+}
+
+// DPT 1.009 OpenClose
+type OpenClose bool
+
+// Pack the datapoint value.
+func (oc OpenClose) Pack() []byte {
+	return packB1(bool(oc))
+}
+
+// Unpack the datapoint value from the given data.
+func (oc *OpenClose) Unpack(data []byte) error {
+	return unpackB1(data, (*bool)(oc))
+}
+
+// String generates a string representation.
+func (oc OpenClose) String() string {
+	if oc {
+		return "Close"
+	}
+	return "Open"
+}
+
+// DPT 1.010 StartStop
+type StartStop bool
+
+// Pack the datapoint value.
+func (ss StartStop) Pack() []byte {
+	return packB1(bool(ss))
+}
+
+// Unpack the datapoint value from the given data.
+func (ss *StartStop) Unpack(data []byte) error {
+	return unpackB1(data, (*bool)(ss))
+}
+
+// String generates a string representation.
+func (ss StartStop) String() string {
+	if ss {
+		return "Start"
+	}
+	return "Stop"
+}
+
+// DPT 5.001  (0% - 100%)
+type Scaling int
```

Comment:
I really dislike `Scaling` as an `int`. Something that is a percentage value should generally be represented as a fraction. Now, since we don't really have fractions in Go, I believe we should pick `float` instead.